lms/djangoapps/teams/views.py

```diff
@@ -313,8 +313,8 @@ def get(self, request):
             )
         else:
             return Response({
-                'developer_message': "unsupported order_by value {}".format(order_by_input),
-                'user_message': _(u"The ordering {} is not supported").format(order_by_input),
+                'developer_message': "unsupported order_by value {ordering}".format(ordering=order_by_input),
+                'user_message': _(u"The ordering {ordering} is not supported").format(ordering=order_by_input),
```

Comment:
ie, what is an "ordering" and what is `{ordering}`? Is it a list of numbers? A person's name? Literally, no idea. Heh.